pal/screen-adapter/web/screen-adapter.ts

```diff
@@ -207,13 +259,19 @@ class ScreenAdapter extends EventTarget {
         this._registerEvent();
     }
 
-    public init (cbToRebuildFrameBuffer: () => void) {
+    public init (configOrientation: number, cbToRebuildFrameBuffer: () => void) {
         this._cbToUpdateFrameBuffer = cbToRebuildFrameBuffer;
-        this._resizeFrame(this._windowSizeInCssPixels);
+        this.orientation = orientationMap[configOrientation];
```

Comment:
From above usage, `auto` string is also used as `configOrientation` parameter, it's not acceptable
1. A parameter should not support two types
2. You haven't declared string type for `configOrientation` parameter
3. Number value shouldn't work at all, because `orientationMap` only have string keys

This issue must be solved

Reply:
the configOrientation should be type of string, here is the change
https://github.com/cocos-creator/engine/pull/9580/commits/8bf757984a6b8971bf28a395b86f18e4783e8a06